packages/backend/src/storage.ts

```diff
@@ -32,7 +32,15 @@ export default (
             next: NextFunction,
             ensName: string,
         ) => {
-            auth(req, res, next, ensName, db, web3Provider, serverSecret);
+            authorize(
```

Comment:
I think the best name for the function would be authMiddleware. Since it is a express middleware intercepting the request https://expressjs.com/en/guide/using-middleware.html

Reply:
I think "auth" in "authMiddleware" would still not clearly state what the function does, while "Middleware" doesn't add much value even though it makes up two thirds of the name. How about "authorizationMiddleware" or just leaving it at "authorize"?